packages/autorest.python/autorest/codegen/models/client.py

```diff
@@ -248,10 +250,19 @@ def has_mixin(self) -> bool:
         return any(o for o in self.operation_groups if o.is_mixin)
 
     @property
-    def has_public_lro_operations(self) -> bool:
+    def has_lro_operations(self) -> bool:
         """Are there any LRO operations in this SDK?"""
         return any(
-            operation.operation_type in ("lro", "lropaging") and not operation.internal
+            is_lro_operation(operation.operation_type)
```

Comment:
I think this would be neater if we just make `Client` have property `lro_operations` which returns a list. Then we can change to property `public_lro_operations` which just calls `self.lro_operations` and returns the public ones. This gets rid of the separate helper function, and represents the relationship between these two properties by having `public_lro_operations` depend on `lro_operations`

Reply:
make sense